icu4c/packaging/distrelease.ps1

```diff
@@ -6,7 +6,7 @@
 # Date: 2017-04-14
 #-------------------------
 #
-# This builds a zipfile containing the *64 bit* Windows binaries.
+# This builds a zipfile containing the 64-bit (Intel x64) and/or 32-bit (Intel x86) Windows binaries.
```

Comment:
super nit: I would suggest removing Intel from the comment here. If you wanted to be really accurate it would be Intel x86 and AMD x86_64 but, probably just simpler to remove trademarked names from the comment.